prow/cmd/deck/main.go

```diff
@@ -922,9 +922,11 @@ lensesLoop:
 	}
 
 	prHistLink := ""
+	prLink := ""
 	org, repo, number, err := sg.RunToPR(src)
 	if err == nil {
 		prHistLink = "/pr-history?org=" + org + "&repo=" + repo + "&pr=" + strconv.Itoa(number)
+		prLink = fmt.Sprintf("https://github.com/%s/%s/pull/%d", org, repo, number)
```

Comment:
This won't work for GitHub enterprise or gerrit. I think we want to use this link instead: https://github.com/kubernetes/test-infra/blob/6a85e1e9718d1d43aaa4b756826b36e8d22eff71/prow/apis/prowjobs/v1/types.go#L592-L593
Ref: https://github.com/kubernetes/test-infra/pull/10891